sdk/loadtestservice/load-testing-rest/samples-dev/sample.ts

```diff
@@ -48,35 +47,42 @@ async function main() {
 
   // Uploading .jmx file to a test
   const fileUploadResult = await client
-    .path("/loadtests/{testId}/files/{fileId}", testCreationResult.body.testId, fileId)
+    .path("/tests/{testId}/files/{fileName}", testId, "sample.jmx")
     .put({
-      contentType: "multipart/form-data",
-      body: {
-        file: readStream,
-      },
+      contentType: "application/octet-stream",
+      body: readStream,
     });
 
   if (isUnexpected(fileUploadResult)) {
     throw fileUploadResult.body.error;
   }
 
-  // Creating app component
+  const fileValidatePoller = await getLongRunningPoller(client, fileUploadResult);
+  const fileValidateResult = await fileValidatePoller.pollUntilDone({
+    abortSignal: AbortController.timeout(60000), // timeout of 60 seconds
+  });
```

Comment:
I remember you saying the operation could take a minute. Should the timeout value be a little more? Also, should we catch the abort error?

Reply:
The operation that we discussed was test run operation which might take longer depending on test duration, anyways I have increased time out for this example as well.